### PR TITLE
Add isLast API to PaginatedResult

### PR DESCRIPTION
**Pull request by gokhanbarisaker**
Resolves: #33

### CODE REVIEW

**Comment by mattheworiordan** [comment]
I would like to see a test please

File: lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelHistoryTest.java [hunk 1]
```diff
@@ -1465,6 +1467,7 @@ public void run() {
 	}
 
 	/**
+<<<<<<< 97c43f880519f2ae4d9b1a160b778c5bca683858
```
Comment (mattheworiordan) [inline lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelHistoryTest.java]: Merge conflict

**Comment by mattheworiordan** [comment]
Thanks @gokhanbarisaker, I didn't realise you were waiting on us, in future please post a message to re-review when you can.  Also, our preferred approach is to [use fixups](http://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html) when updating an already up PR. This way we can review the fixes you submit, and when ready, you can rebase by squashing the commits before merge.  Is that OK?

**Comment by gokhanbarisaker** [comment]
I will start doing as instructed :)

**Comment by paddybyers** [comment]
LGTM

**Pull request merged**